### PR TITLE
TypeStatus and Sex vocabularies

### PR DESCRIPTION
**Pull request by marcos-lg**
This vocabulary is a different case than the others since the typeStatus term is multivalue and we flag the records with issues.

### CODE REVIEW

**Comment by marcos-lg** [comment]
I converted this PR to draft because I'm gonna integrate the Sex vocabulary in this branch too. It shouldn't require many changes.

**Comment by marcos-lg** [comment]
I added the Sex vocabulary and it's ready to review.

File: sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java [hunk 1]
```diff
@@ -18,6 +25,10 @@
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class VocabularyInterpreter {
 
+  /** Values taken from <a href="https://github.com/gbif/vocabulary/issues/87">here</a> */
+  private static final List<String> SUSPECTED_TYPE_STATUS_VALUES =
```
Comment (muttcg) [inline sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java:29]: Why it is not - Set.of()?

File: sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java [hunk 2]
```diff
@@ -55,21 +66,74 @@ public static BiConsumer<ExtendedRecord, EventCoreRecord> interpretEventType(
...
+    return (er, br) ->
+        extractListValue(er, DwcTerm.typeStatus)
+            .forEach(
+                value ->
+                    interpretVocabulary(
+                            DwcTerm.typeStatus,
+                            value,
+                            vocabularyService,
+                            v -> {
+                              if (SUSPECTED_TYPE_STATUS_VALUES.stream()
```
Comment (muttcg) [inline sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java:81]: SUSPECTED_TYPE_STATUS_VALUES.contains(v.toLowerCase())

**Review by muttcg — CHANGES_REQUESTED** [review]
Need a small change in one place

File: sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java (hunk 2 above)
Comment (muttcg) [reply sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java:81]: Or v is some long text?

File: sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java (hunk 1 above)
Comment (marcos-lg) [reply sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java]: Just changed it to Set.of

File: sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java (hunk 2 above)
Comment (marcos-lg) [reply sdks/core/src/main/java/org/gbif/pipelines/core/interpreters/core/VocabularyInterpreter.java:80]: yeah, v can be a long text(e.g.: possibly allotype)

**Review by muttcg — APPROVED** [review]

**Pull request merged**